src/main/java/core/basesyntax/dao/FruitShopDao.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface FruitShopDao {
+    List<FruitTransaction> parseTransactions(String dataFromFile);
```

Comment:
Должны ли мы лезть в этот интервейс и изменять имя переменной, если заказчик начнет присылать данные не в файле, а как-то по другому? Важно ли этому интерфейсу, откуда получены данные? Помни об **АБСТРАКЦИИ**.
```suggestion
    List<FruitTransaction> parseTransactions(String data);
```